docker/webui/run_openqa.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# prepare environment
+rm -f /run/dbus/pid
+/usr/share/openqa/script/upgradedb --user geekotest --upgrade_database
+
+# run services
+dbus-daemon --system --fork
+su geekotest -c '/usr/share/openqa/script/openqa-scheduler &'
+su geekotest -c '/usr/share/openqa/script/openqa-websockets &'
+su geekotest -c '/usr/share/openqa/script/openqa gru -m production run &'
+rcapache2 start && su geekotest -c '/usr/share/openqa/script/openqa prefork -m production --proxy'
```

Comment:
It's possible to start systemd in Docker, but I would say that it isn't really all that clean. Docker is usually used so that it runs only one process per each container. When you need to run more processes as a part of one application, you create one container for each process and then link them together so that they could communicate with each other via network, or create shared storage volume for them. For example, if you have a web app, you have one container for app itself, one container for Postgres, one container for Redis, then you link Postgres and Redis containers to app container and only expose app container itself to the outside world.

In this example, I don't think that it would make sense to separate scheduler, websocket server and openQA itself into separate containers (although I think that it would be /the Docker way/), but I would still hesitate to start systemd in it (even though it would be possible). 

Think of Docker like some kind of "chroot for single process".

I'm of course opened for discussion and while I think that running systemd inside Docker would be kind of hackish, if you would want it, I would change it appropriately.